api-docs/generator.py

```diff
@@ -183,7 +183,6 @@ def cli(output_path):
 
         vars = {
             'org': org,
-            'api_key': api_key,
```

Comment:
I just happened to notice this was repeated, since I started initially looking through doc generation since that's how the bug was first brought to my attention.